### PR TITLE
create arabic list words page related with #23

### PR DESCRIPTION
**Pull request by SajaLahaleeh**

### CODE REVIEW

File: client/src/components/arabicListWords/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import axios from 'axios'
+import './style.css'
+
+class arabicListWords extends React.Component {
+  state = {
+    arabicList: []
+  }
+
+  componentDidMount() {
+    axios.get(`api/arabic-words`).then(result => {
```
Comment (jema28) [inline client/src/components/arabicListWords/index.js]: You can destructure result here in the parameter i.e. `({data} => `

File: client/src/components/arabicListWords/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      this.setState({ arabicList: result.data })
+      console.log('the result:', result.data)
```
Comment (jema28) [inline client/src/components/arabicListWords/index.js]: Please delete this.

File: client/src/components/arabicListWords/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    })
+  }
+
+  render() {
+    return (
+      <>
+        <h1> List Of Arabic Words </h1>
+        <React.Fragment>
```
Comment (jema28) [inline client/src/components/arabicListWords/index.js]: Change this to <></> to keep consistent or a div as you're going to add styles.

File: client/src/components/arabicListWords/style.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
+.word1 {
+  position: absolute;
```
Comment (jema28) [inline client/src/components/arabicListWords/style.css]: Why are you using absolute positioning here? Check this on all of them.

File: server/controllers/index.js [hunk 5]
```diff
@@ -4,9 +4,11 @@ const { getPlaces } = require('./places')
 const { login } = require('./login')
 const { getGuides } = require('./guides')
 const { isAuthenticated } = require('../middleware/auth')
+const { getArabicWords } = require('./arabicList')
 
 router.get('/places', isAuthenticated, getPlaces)
 router.post('/login', login)
 router.get('/guides', getGuides)
+router.get('/arabic-words', getArabicWords)
```
Comment (jema28) [inline server/controllers/index.js]: Does someone need to be authenticated to see this?

File: server/database/queries/getArabicWords.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const dbConnention = require('../dbConnection')
+
+const getArabicList = () => {
+  return dbConnention.query('SELECT * FROM arabic_words;').then(res => res.rows)
```
Comment (jema28) [inline server/database/queries/getArabicWords.js]: You need to add a catch here.

File: client/src/components/arabicListWords/style.css (hunk 4 above)
Comment (SajaLahaleeh) [reply client/src/components/arabicListWords/style.css]: for CSS I want to work it today on my home.

File: server/controllers/index.js (hunk 5 above)
Comment (SajaLahaleeh) [reply server/controllers/index.js]: No, just we need an authentication when the tourist want to add a review for a guides or places.

File: client/src/App.js [hunk 7]
```diff
@@ -7,6 +7,7 @@ import Guides from './components/Guides'
 import OnePlace from './components/OnePlace'
 import Login from './components/SharedComponents/Login'
 import NavBar from './components/SharedComponents/navbar'
+import arabicListWords from './components/arabicListWords/index'
```
Comment (jema28) [inline client/src/App.js]: Please delete index here.

File: client/src/components/arabicListWords/index.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+      this.setState({ arabicList: data })
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <h1> List Of Arabic Words </h1>
+        <React.Fragment>
+          {this.state.arabicList.map(element => (
```
Comment (jema28) [inline client/src/components/arabicListWords/index.js]: You can destructure element here also in the parameter as you're repeating this below.

File: client/src/components/arabicListWords/index.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+            <ul>
+              <li className="list">
+                <div>
```
Comment (jema28) [inline client/src/components/arabicListWords/index.js]: I'm confused about this structure, why are you making a ul ordered list and then divs? I think you just need to create divs here.

File: client/src/components/arabicListWords/style.css (hunk 4 above)
Comment (jema28) [reply client/src/components/arabicListWords/style.css]: Okay, I don't want to merge it like this though. Can you at least delete the absolute positioning?

File: client/src/components/arabicListWords/index.js (hunk 9 above)
Comment (SajaLahaleeh) [reply client/src/components/arabicListWords/index.js]: Ok, but I want to think how I need to display the list so when I make a CSS I solve this.

File: client/src/components/arabicListWords/style.css [hunk 10]
```diff
@@ -0,0 +1,40 @@
+.word1 {
```
Comment (jema28) [inline client/src/components/arabicListWords/style.css]: I would delete this whole file and start the css from scratch.

**Pull request merged**